frontend/src/app/shared/components/storages/file-picker-modal/file-picker-modal.component.html

```diff
@@ -1,5 +1,5 @@
 <div
-  class="spot-modal spot-modal_wide op-file-picker"
+  class="spot-modal op-file-picker"
```

Comment:
Sure we don't need ` wide`  here?

Reply:
Marc reduced the file picker to the normal modal width.